service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/DocumentMerger.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.fpl.service.orders.generator;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.service.DocumentDownloadService;
+import uk.gov.hmcts.reform.fpl.service.docmosis.DocumentConversionService;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.jsonwebtoken.lang.Collections.isEmpty;
+import static org.apache.pdfbox.io.MemoryUsageSetting.setupMainMemoryOnly;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class DocumentMerger {
+
+    private final DocumentDownloadService documentDownloadService;
+    private final DocumentConversionService documentConversionService;
+
+    public DocmosisDocument mergeDocuments(
+        DocmosisDocument originalDocument, List<DocumentReference> additionalDocuments) {
```

Comment:
vv minor but if you have the method declaration like so
```java
    public DocmosisDocument mergeDocuments(DocmosisDocument originalDocument,
                                           List<DocumentReference> additionalDocuments) {
```
it looks better in my opinion